wiki/Game_Modes/en.md

```diff
@@ -21,7 +21,7 @@ Based on *[Taiko no Tatsujin](https://en.wikipedia.org/wiki/Taiko_no_Tatsujin "W
 
 *Main page: [osu!catch](osu!catch)*
 
-Collect falling fruits by moving the catcher with a plate. Keep dancing and do not miss the step of the beat.
+Based on the EZ2CATCH mode from the *[EZ2DJ series](https://ja.wikipedia.org/wiki/Ez2Dj/EZ2AC "Wikipedia")*. Collect falling fruits by moving the catcher with a plate. Keep dancing and do not miss the step of the beat!
```

Comment:
```suggestion
Based on the EZ2CATCH mode from the *[EZ2DJ series](https://en.wikipedia.org/wiki/EZ2DJ "Wikipedia")*. Collect falling fruits by moving the catcher with a plate. Keep dancing and do not miss the step of the beat!
```
Should link to the English article.

Reply:
I would, but the English article doesn't mention anything about EZ2CATCH

Reply:
it doesn't look like there are any good resources that talk about EZ2CATCH in English, and Japanese is even less helpful so I've just left the reference to EZ2DJ as the English Wikipedia article

not much we can do about that short of writing something ourselves. which probably isn't necessary